js/private/js_run_devserver.mjs

```diff
@@ -111,15 +196,36 @@ async function syncRecursive(src, dst, writePerm) {
                             await syncRecursive(
                                 path.join(src, entry),
                                 path.join(dst, entry),
+                                sandbox,
                                 writePerm
                             )
                     )
                 )
             ).reduce((s, t) => s + t, 0)
         } else {
+            if (lstat.size < MAX_CHECKSUM_FILESIZE) {
```

Comment:
Did you do any experiments with this constant? I guess the IO of reading the file is much less then IO of writing to the devserver dir?

Reply:
I didn't. On 2nd thought, this check isn't that useful so I'll remove it. If a 1p package has such a large file then it comes with many other problems and not just some additional latency on the md5 checksum.

Reply:
If we aren't sure then I'd rather drop the code and we can come back to it if we ever notice an issue. Keep it simple until we find an issue with it.